packages/react-experiments/src/components/Pagination/PageNumber.tsx

```diff
@@ -14,9 +14,10 @@ export class PageNumber extends React.Component<IPageNumberProps, {}> {
       <DefaultButton
         key={page}
         onClick={this._onClick}
-        aria-selected={selected}
+        aria-checked={selected}
         aria-label={ariaLabel}
         styles={{ root: className }}
+        role="radio"
```

Comment:
The [MDN documentation for the radio role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role) says they must be grouped inside an element with role="radiogroup". I couldn't immediately verify that's the case--do you need to add that to the parent element?

Reply:
Sure, will raise a separate PR for that!